.travis.yml

```diff
@@ -1,13 +1,13 @@
 language: python
 required: sudo
 python:
-    "2.7"
+    - "2.7"
+    - "3.2"
 virtualenv:
     system_site_packages: true
 install:
-  - sudo apt-get install python-opencv
+  - sudo -H python -m pip install opencv-python pillow
```

Comment:
I was going to say we should have both CV2 for both python 2 and 3, but realize there's no package for python3-opencv :( In this case, I'd say we can compile cv2 from source for both python2/3, or do cv2 in python2 and cv3 in python3. This seems to be doable using a build matrix
https://docs.travis-ci.com/user/customizing-the-build/#Explicitly-Including-Jobs